components/splitbutton/stories/template.js

```diff
@@ -0,0 +1,71 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { when } from "lit-html/directives/when.js";
+
+import { Template as Button } from '@spectrum-css/button/stories/template.js';
+
+import "../index.css";
+
+export const Template = ({
+  rootClass = "spectrum-SplitButton",
+  customClasses = [],
+  size = "m",
+  variant = "cta",
+  iconName = "ChevronDown100",
+  position = "left",
+  ...globals
+}) => {
+
+  return html`
+    ${when(position === "right", () =>
```

Comment:
The markup for these 2 when statements seems really similar - is there a way to combine them?